### PR TITLE
Updated WPScan.tsx with new UI and parameters

### PR DESCRIPTION
**Pull request by chandelshubham63**
This pull req solves the issue #939. I have improved the **WPScan tool interface.** Maintained existing functionality while adding some more crucial specific parameters (e.g., random user agent, force SSL, proxy settings). Reorganized options into collapsible sections and implemented a three-step process using the Stepper component. This enhancement aims to improve usability for users and maintain a consistent look in the toolkit.

Testing of this tool has been completed, and it is working perfectly as expected.

### CODE REVIEW

**Comment by chandelshubham63** [comment]
@michaeljpigott  I have thoroughly tested the tool and checked the format, so you shouldn’t encounter any issues. It is ready to be merged into the main branch :)

File: src/components/WPScan/WPScan.tsx [hunk 1]
```diff
@@ -1,116 +1,107 @@
...
+    const title = "WPScan";
+    const description = "WPScan scans remote WordPress installations to find security issues";
     const steps =
         "Step 1: Enter a WordPress URL.\n" +
-        "Step 2: Enter any additional options for the scan.\n" +
-        "Step 3: Enter any additional parameters for the scan.\n" +
-        "Step 4: Click Scan to commence WPScan's operation.\n" +
-        "Step 5: View the Output block below to view the results of the tool's execution.\n" + //Steps to run the component
+        "Step 2: Enter any additional parameters for the scan.\n" +
+        "Step 3: Click Scan to commence WPScan's operation.\n" +
```
Comment (michaeljpigott) [inline src/components/WPScan/WPScan.tsx:63]: The convention is Start + title of tool. 
```suggestion
        "Step 3: Click Start " + title + " to commence WPScan's operation.\n" +
```

File: src/components/WPScan/WPScan.tsx [hunk 2]
```diff
@@ -120,285 +111,258 @@ const WPScan = () => {
...
+                                            {...form.getInputProps("outputFormat")}
+                                        />
+                                        <TextInput label="Output File" {...form.getInputProps("outputFile")} />
+                                    </Stack>
+                                )}
+                            </Stepper.Step>
+                            <Stepper.Step label="Run">
+                                <Stack align="center" mt={20}>
+                                    <Button type="submit" disabled={loading} style={{ alignSelf: "center" }}>
+                                        Run WPScan
```
Comment (michaeljpigott) [inline src/components/WPScan/WPScan.tsx:354]: The convention is Start + title of tool. 
```suggestion
                                        Start {title}
```

**Review by michaeljpigott — CHANGES_REQUESTED** [review]
Hi @chandelshubham63 

Looks good. Just a couple of small suggestions. I did not test the tool because I do not have a WordPress website running ATM. 

Kind regards

Michael

**Comment by chandelshubham63** [comment]
@michaeljpigott  all requested changes completed, ready to merge thanks:)

**Review by michaeljpigott — APPROVED** [review]
Hi @chandelshubham63 

Thank you for making those changes. I approved this pull request. 

Kind regards

Michael

**Pull request merged**